css/src/_m-message.scss

```diff
@@ -0,0 +1,72 @@
+.m-messages {
+    padding: 0;
+    font-style: italic;
+    list-style: none;
+
+    &__item {
+        --message-txt: #da394c;
+        --message-bg: rgba(238, 162, 171, 0.25);
+        position: relative;
```

Comment:
Add empty line between css variables and properties.

Reply:
Added an empty line between css variables and properties.